src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -95,6 +104,10 @@ public GeoApiContext() {
       }
     }
 
+    if(channelSet==false && channel != null && !channel.isEmpty() ){
+      query.append("&chanel=").append(channel);
```

Comment:
Also, please reformat the if statement :)

`if (channelSet == false && channel != null && !channel.isEmpty()) {`